### PR TITLE
commit to branch supraja

### PR DESCRIPTION
**Pull request by vempalisupraja**
Testing pull request. Added some abstract methods to IAdminDAO.

### CODE REVIEW

File: src/com/cap/omts/dao/AdminDAOImpl.java [hunk 1]
```diff
@@ -51,7 +50,7 @@
 	@Override
 	public int addTheatre(Theatre theatre) throws OMTSException {
 		// TODO Auto-generated method stub
-		int insertedRows;
```
Comment (PLaxmiPrasanna) [inline src/com/cap/omts/dao/AdminDAOImpl.java]: please rename the variable appropriately. Here I suggest you to rename it as isInserted and initialize it to zero.

File: src/com/cap/omts/dao/AdminDAOImpl.java [hunk 2]
```diff
@@ -72,7 +69,7 @@ public int addTheatre(Theatre theatre) throws OMTSException {
 					throw new OMTSException("problem while closing Database");
 				}
```
Comment (PLaxmiPrasanna) [inline src/com/cap/omts/dao/AdminDAOImpl.java:70]: close the database connection

File: src/com/cap/omts/dao/AdminDAOImpl.java [hunk 3]
```diff
@@ -72,7 +69,7 @@ public int addTheatre(Theatre theatre) throws OMTSException {
 					throw new OMTSException("problem while closing Database");
 				}
 			}
-		return insertedRows;
+		return rows;
 	}
```
Comment (PLaxmiPrasanna) [inline src/com/cap/omts/dao/AdminDAOImpl.java:75]: please change the insertedRows variable to an appropriate name. I suggest you to name it as isInserted and initialise it.

File: src/com/cap/omts/dao/AdminDAOImpl.java [hunk 4]
```diff
@@ -50,10 +47,43 @@
...
+	
 	
 	/*
 	 * public static void main(String[] args) throws OMTSException {
-	 * System.out.println(AdminDAOImpl.getTheatreDetails()); }
+	 * System.out.println(AdminDAOImpl.getTheatreDetails());
+	 * System.out.println(AdminDAOImpl.addTheatre(new
+	 * Theatre(123,"Game","Tokyo","James","9028384756"))); }
 	 */
+
```
Comment (PLaxmiPrasanna) [inline src/com/cap/omts/dao/AdminDAOImpl.java]: Please remove this comment

File: src/com/cap/omts/dao/AdminDAOImpl.java [hunk 5]
```diff
@@ -1,47 +1,58 @@
...
+			resultSet = statement.executeQuery("SELECT * FROM THEATRE");  
+				while(resultSet.next())   {
+					Theatre theatre = new Theatre();
+					theatre.setTheatreId(resultSet.getInt(1));
+					theatre.setTheatreName(resultSet.getString(2));
+					theatre.setTheatreCity(resultSet.getString(3));
+					theatre.setManagerName(resultSet.getString(4));
+					theatre.setManagerContact(resultSet.getString(5));
+					theatreList.add(theatre);
+				System.out.println(resultSet.getInt(1)+"  "+resultSet.getString(2)+"  "+resultSet.getString(3)+" "+resultSet.getString(4)+" "+resultSet.getString(5));
```
Comment (PLaxmiPrasanna) [inline src/com/cap/omts/dao/AdminDAOImpl.java]: please remove this syso statement

File: src/com/cap/omts/dao/AdminDAOImpl.java [hunk 6]
```diff
@@ -1,47 +1,58 @@
...
+				}
+			}
+				
+		return theatreList;
+		}
 	
+	/*
+	 * public static void main(String[] args) throws OMTSException {
+	 * System.out.println(AdminDAOImpl.getTheatreDetails()); }
+	 */
```
Comment (PLaxmiPrasanna) [inline src/com/cap/omts/dao/AdminDAOImpl.java]: please remove comments

**Review by PLaxmiPrasanna — COMMENTED** [review]
please do the changes as commented on each part.

**Review by PLaxmiPrasanna — CHANGES_REQUESTED** [review]
please do the mentioned changes

**Comment by vempalisupraja** [comment]
The changes requested are reflected.

**Pull request closed without merging**